portal-api/src/constants/deal.js

```diff
@@ -62,4 +68,5 @@ module.exports = {
   SUBMISSION_TYPE,
   ACTION_NAME,
   DEAL_STATUS,
+  SUBMISSION_TYPE_LOWERCASE,
```

Comment:
where is this being used?

Reply:
removed, I was going to use this elsewhere but forgot to remove it